packages/roosterjs-editor-dom/lib/table/pasteTable.ts

```diff
@@ -48,12 +48,21 @@ export default function pasteTable(
         for (let j = cursorCol; j < columns; j++) {
             let cell = currentTable.getCell(i, j);
             let newCell = newTable.getTd(i - cursorRow, j - cursorCol);
-            if (cell.td) {
-                moveChildNodes(cell.td, newCell!);
+            if (cell.td && newCell) {
+                moveChildNodes(cell.td, newCell);
+                copyCellStyle(cell.td, newCell);
             } else {
                 cell.td = document.createElement('td');
             }
         }
     }
-    currentTable.writeBack();
+
+    currentTable.writeBack(true /* skipApplyFormat*/);
```

Comment:
What if I apply format later, say I pasted a table into a table, then I insert a row or column, what will happen?

Reply:
I'm using saveMetadata with  bgColorOverride: true, then write applyFormat will not apply format to this cells.